### DISCUSSION TITLE
How to get network error type in javascript

### DISCUSSION THREAD

**Discussion by Eboubaker** [livewire/livewire#3806] [1. Help] [answered]
i want to show a flash message for the user when the response from the server is 500 or 400 and get the exception message.
i tried this but it logged `undefined`
```javascript
window.Livewire.onError((err) => console.log(err))
```

**Comment by lucasromanojf** [comment, marked as answer]
PR #4179 merged

**Reply by Eboubaker** [reply]
Thank you, your pr fixed it.
